x-pack/plugins/infra/public/components/metrics_explorer/helpers/create_tsvb_link.ts

```diff
@@ -88,9 +88,9 @@ const createFilterFromOptions = (
     filters.push(options.filterQuery);
   }
   if (options.groupBy) {
-    filters.push(`${options.groupBy}: ${series.id}`);
+    filters.push(`${options.groupBy} : "${series.id}"`);
```

Comment:
There is a non-zero risk here that the field names or the id contain characters that need escaping, such as quotes. Generating the KQL expression from an AST could work around that. Not sure if the risk is sufficient to justify the effort though, so I would leave that up to your judgement.